plugins/push/frontend/public/stylesheets/main.css

```diff
@@ -987,7 +1015,7 @@ table.d-table .nothing { opacity: 0.3; }
 
 .comp-push input[type="number"]::-webkit-inner-spin-button,
 .comp-push input[type="number"]::-webkit-outer-spin-button {
-    -webkit-appearance:none;
```

Comment:
Ok, I searched for other input-number related to .com-push class and I didn't find. 
But you're right. I can add a specific class for those inputs.